python/cog/server/http.py

```diff
@@ -383,12 +504,18 @@ def _predict(
 
         # ...and return the result.
         try:
-            response = PredictionResponse(**predict_task.result.dict())
+            response_object = (
+                unwrap_pydantic_serialization_iterators(
```

Comment:
Sorry if I'm being slow here, but have you written anywhere about why this unwrapping is needed? I would kind of expect that `model_dump` should do the right thing?

Reply:
Alright, just pushed a commit documenting that method and updated the PR with details.